src/Pages/ProductList/ProductListMain/MainContents/ProductListMainFooter/ProductListPageBtn/ProductListPageBtn.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import mixin from "../../../../../../Styles/mixin";
+
+export default class ProductListPageBtn extends Component {
+  render() {
+    const { pageNum, pageNumClickedAt, clickHandlerBtnClicked } = this.props;
+    return (
+      <ProductListPageBtnContainer
+        pageNum={pageNum}
+        pageNumClickedAt={pageNumClickedAt}
+        onClick={() => clickHandlerBtnClicked("pageNum", pageNum)}
+      >
+        {pageNum}
+      </ProductListPageBtnContainer>
+    );
+  }
+}
+
+const ProductListPageBtnContainer = styled.div`
+  ${mixin.center};
+  width: 32px;
+  height: 32px;
+  margin: 0 10px;
+  border-radius: 50%;
+  background-color: ${({ pageNum, pageNumClickedAt }) =>
+    pageNum === pageNumClickedAt ? "black" : "white"};
```

Comment:
```suggestion
  background-color: ${({ isSameIdx }) => isSameIdx ? "black" : "white"};
```
불필요한 코드 중복을 줄여줄 수 있는 방법을 고민해주세요

Reply:
넵 수정완료